Dockerfile

```diff
@@ -1,25 +1,43 @@
 FROM debian:buster-slim
 
 # File Author / Maintainer
-MAINTAINER AGSPhoenix
+LABEL maintainer=AGSPhoenix
+
+ARG DEBIAN_FRONTEND=noninteractive
+# Makes the apt-get process aware
+# of the docker build process env
+# so it doesn't complain as much
 
 RUN \
 # MAN folder needed for jre install
-     mkdir -p /usr/share/man/man1 \
-  && mkdir -p /sheep/cache \
-# Install JRE and curl
-  && apt-get update \
-  && apt-get install -y --no-install-recommends \
+	mkdir -p \
+		/usr/share/man/man1 \
+		/sheep/cache \
+	# Install JRE, curl and blender deps
+	&& apt-get update \
+	&& apt-get upgrade -y \
+	&& apt-get install -y --no-install-recommends \
 	default-jre-headless \
 	curl \
-	#Blender dependencies
-	libsdl1.2debian \
-	libxxf86vm1 \
-	libgl1-mesa-glx \
 	libglu1-mesa \
-	libxi6 \
- 	libxrender1 \
-	libxfixes3
+	# Blender dependencies of other sheepit containers explained:
+	# needed in the past:
```

Comment:
Have you verified these aren't necessary for at least projects running the oldest version of Blender that the farm supports (preferably all versions, but I'm not that picky)?

Reply:
Yes, I have, the oldest blender version doesn't fail, my fork has been running this configuration since 15 Feb 2020 on my fork, but just in case I did one under Blender 2.79b on the farm.
![image](https://user-images.githubusercontent.com/2771553/110219148-cd1bc500-7ebd-11eb-9520-b06ec6627c8a.png)
![image](https://user-images.githubusercontent.com/2771553/110219171-ee7cb100-7ebd-11eb-95ef-960edba7b387.png)
Works. 👍